### ISSUE TITLE
logsEnabled not working

### ISSUE THREAD

**Issue by LautaroLorenz** [tnicola/ngx-joyride#59]
hello, i set

```
    this._joyrideService.startTour({
      logsEnabled: false,
      steps: [
        'mapa',
        'herramientas',
        'hFechaPosiciones',
        'hCentrarMapa',
        'hZoomMapa',
        'hAliasNombre',
        'hGeoMenu',
      ]
    }).subscribe(
      (step) => { },
      (error) => { },
      () => { }
    );
```

but on console i can see

> ngx-joyride.esm.js:110 ngx-joyride:::Adding step hFechaPosiciones to the steps list. 
> 12:44:17.116 ngx-joyride.esm.js:110 ngx-joyride:::Adding step hCentrarMapa to the steps list. 
> 12:44:17.117 ngx-joyride.esm.js:110 ngx-joyride:::Adding step hZoomMapa to the steps list. 
> 12:44:17.124 ngx-joyride.esm.js:110 ngx-joyride:::Adding step hAliasNombre to the steps list. 
> 12:44:17.126 ngx-joyride.esm.js:110 ngx-joyride:::Adding step hGeoMenu to the steps list. 
> 12:44:17.128 ngx-joyride.esm.js:110 ngx-joyride:::Adding step herramientas to the steps list. 
> 12:44:17.129 ngx-joyride.esm.js:110 ngx-joyride:::Adding step mapa to the steps list. 

I expected not to see any message in console

**Comment by mathiamo** [comment]
Same issue here, any update on this?

**Comment by sangeethjohn01** [comment]
any update?

**Comment by SharadaRamji** [comment]
Hai @tnicola. Can you please look into the issue and update. Thank you in advance.

**Comment by vinh28193** [comment]
logsEnabled should have be default is false :)

**Comment by SharadaRamji** [comment]
The default value is false but it not working as it should be.
Since the default functionality is not working. I have explicitly mentioned the same in my code
`this.joyrideService.startTour(
            {
                **logsEnabled: false,**
                steps,
                showCounter: false
            }
        );`
Still the console displays the logs
![image](https://user-images.githubusercontent.com/32533849/66739926-75775680-ee8f-11e9-982f-dd7fe074ac6b.png)

**Comment by SharadaRamji** [comment]
The default value is false but it not working as it should be.
Since the default functionality is not working. I have explicitly mentioned the same in my code
`this.joyrideService.startTour(
            {
                logsEnabled: false,
                steps,
                showCounter: false
            }
        );`
Still the console displays the logs
![image](https://user-images.githubusercontent.com/32533849/66739926-75775680-ee8f-11e9-982f-dd7fe074ac6b.png)

**Comment by tazilios** [comment]
Hi @tnicola ,

It's difficult for me to deploy a production application showing those logs. 
Would you mind having a look?

Best regards

**Comment by sangeethjohn01** [comment]
**any update**?

**Comment by DMZakaria** [comment]
any update ?

**Comment by SharadaRamji** [comment]
Any Update????????

**Comment by DMZakaria** [comment]
Guys, I did a quick workaround, I know it is not the best solution!
In your main.ts you can disable the console.info, do the following :
 
`if (environment.production) {
  enableProdMode();
  disableConsole();
}
........
function disableConsole() {
  const newConsole = {} as any;
  newConsole.info = () => {
    //
  };`

**Comment by Talpi208** [comment]
Any update????

**Comment by Talpi208** [comment]
> Hi @tnicola ,
> 
> It's difficult for me to deploy a production application showing those logs.
> Would you mind having a look?
> 
> Best regards

Did you find a solution?

**Comment by tnicola** [comment]
Hi all, I'm really sorry for the late reply. I'm working on it.

**Comment by tnicola** [comment]
It should be solved in the new version 2.2.11
